### PR TITLE
 Small documentation improvements

### PR DESCRIPTION
**Pull request by Anchakor**
I picked up Godot to give it a try for Ludum Dare 44, it was a good experience overall. 
These are some documentation improvements which would have helped me.

### CODE REVIEW

**Comment by Jummit** [comment]
I think squashing the commits is the required for this to be merged.

**Comment by Anchakor** [comment]
> I think squashing the commits is the required for this to be merged.

Should I do that (on fork master then do force push)? Or can I leave it to the maintainers (AFAIK they can do that on GitHub)?

**Comment by YeldhamDev** [comment]
@Anchakor It's not hard to do it by yourself:
https://docs.godotengine.org/en/3.1/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

**Comment by Anchakor** [comment]
@YeldhamDev thanks for the link, done now I think

File: doc/classes/Path2D.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Can have [PathFollow2D] child-nodes moving along the [Curve2D]. See [PathFollow2D] for more information on this usage.
+		Note that the path is considered as relative to the moved nodes (children of [PathFollow2D]) - usually the curve should start with a zero vector (0,0).
```
Comment (YeldhamDev) [inline doc/classes/Path2D.xml]: Add a space after the comma in "(0,0)".

File: doc/classes/Vector2.xml [hunk 2]
```diff
@@ -216,7 +216,7 @@
 			<argument index="0" name="phi" type="float">
 			</argument>
 			<description>
-				Returns the vector rotated by [code]phi[/code] radians.
+				Returns the vector rotated by [code]phi[/code] radians. See also [code]deg2rad()[/code].
```
Comment (YeldhamDev) [inline doc/classes/Vector2.xml]: Write it as "[method @GDScript.deg2rad]" so it actually links to the method.

**Comment by mhilbrunner** [comment]
Looks good to me. Thanks for this, and congrats on your first contribution to Godot! :)

**Comment by Anchakor** [comment]
@mhilbrunner @YeldhamDev  Thanks of maintainership and review!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**